tests/snuba/api/endpoints/test_organization_events_mep.py

```diff
@@ -3134,15 +3134,8 @@ class OrganizationEventsMetricsEnhancedPerformanceEndpointTestWithOnDemandMetric
 
     def setUp(self) -> None:
         super().setUp()
-
-    def do_request(self, query: Any) -> Any:
-        self.login_as(user=self.user)
-        url = reverse(
-            self.viewname,
-            kwargs={"organization_slug": self.organization.slug},
-        )
-        with self.feature({"organizations:on-demand-metrics-extraction-widgets": True}):
-            return self.client.get(url, query, format="json")
+        self.url = reverse(self.viewname, kwargs={"organization_slug": self.organization.slug})
+        self.features = {"organizations:on-demand-metrics-extraction-widgets": True}
```

Comment:
These two parameters will be used within `_do_request` in the parental class.
<img width="497" alt="image" src="https://github.com/getsentry/sentry/assets/44410/37e33c35-bd50-4b29-8e21-f6b56f01b7d1">